### DISCUSSION TITLE
Running nanoVNASaver in Eclipse - getting "DEBUG" messages from where / how ?

### DISCUSSION THREAD

**Discussion by anneranch** [NanoVNA-Saver/nanovna-saver#646] [Q&A]
I have managed to RUN->Run  my cloned nanoVNASaver 
AFTER I got the main window and just selected "connect" I dis minimize the window and observed
![Screenshot from 2023-06-21 13-48-21](https://github.com/NanoVNA-Saver/nanovna-saver/assets/48501240/1a859752-0bb9-4af6-8846-b1a81752715b)
 the Eclipse messages.  Majority of them were prefixed with "DEBUG" word. Where in the  Python code is that implemented ?

**Comment by anneranch** [comment]
Since I cannot figure out how to edit the original post - here is an update 

I managed to add "tool tip"  to one button...
I am ready to do some serious work...

BTW it looks as the reason "update" does not "connect" -  it looks as it is connecting to update nanoVNA  firmware .
I was under the erroneous impression it was connecting to update nanoVNASaver. 

These are obviously  TWO different pieces of software

**Reply by Ho-Ro** [reply]
> Since I cannot figure out how to edit the original post ...

Just click the three dots

![image](https://github.com/NanoVNA-Saver/nanovna-saver/assets/12542359/a8a686aa-8eb1-4cab-8ae2-173a2c49916e)

and select Edit

![image](https://github.com/NanoVNA-Saver/nanovna-saver/assets/12542359/83737ff7-28c9-4e63-8a29-4836b23d9a74)
